### PR TITLE
ci: upgrade node version

### PR DESCRIPTION
**Pull request by baabouj**
use Node.js version 18 in CI/CD pipelines.

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
+          node-version: "18.x"
```
Comment (dai-shi) [inline .github/workflows/cd.yml:17]: Lately, I prefer this style for no reason. 
```suggestion
          node-version: 18
```

Can you also upgrade `jsdom` (I assume node 14 was restricting.)
I will also change the Vercel deploy config later.

Comment (baabouj) [reply .github/workflows/cd.yml:17]: Got it 👍😆

**Review by dai-shi — APPROVED** [review]
LGTM

**Comment by dai-shi** [comment]
image

**Pull request merged**